### ISSUE TITLE
Cannot access the examples in latest update

### ISSUE THREAD

**Issue by zfields** [microsoft/vscode-arduino#1363]
- Press F1
- Select "Arduino: Examples"
- An empty tab appears on Linux

Downgrading fixed this issue.

**Comment by barthez** [comment]
Same issue here. v0.4.6 makes examples, boards and programmers list disappear. Downgrading to 0.4.5 makes it work again. I'm using macOS BigSur and arduino-cli:
```
    "arduino.logLevel": "verbose",
    "arduino.ignoreBoards": [

    ],
    "arduino.useArduinoCli": true,
    "arduino.path": "/usr/local/bin",
    "arduino.commandPath": "arduino-cli",
    "arduino.additionalUrls": ""
```

**Comment by benmcmorran** [comment]
Fixed in v0.4.7.
